examples/time/time.bal

```diff
@@ -2,7 +2,7 @@ import ballerina/io;
 import ballerina/time;
 
 public function main() returns error? {
-    // Get the current instant of the system clock(seconds from the epoch of
+    // Gets the current instant of the system clock(seconds from the epoch of
     // 1970-01-01T00:00:00). The returned `time:Utc` value represents seconds
     // from the epoch with nanoseconds precision.
     // The `time:Utc` is a tuple with [int, decimal]. The first member of the
```

Comment:
```suggestion
    // The `time:Utc` is a tuple with `[int, decimal]`. The first member of the
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383